software/mk/Makefile.verilog

```diff
@@ -57,6 +57,10 @@ $(info Machine executable: $(BSG_SIM_EXE))
 
 %.run: NBF_FILE=$*.nbf
 %.run: %.nbf
+# Rebuid the machine if machines dir is internal
```

Comment:
@vb000 I don't fully understand the purpose of this, and I suspect there might be a cleaner way to do it. Can you explain it to me? Perhaps there needs to be a dependence on $(BSG_SIM_EXE)?

Reply:
BSG_SIM_EXE can be in an external repo like bsg_designs. This if make sure to rebuild only when machines path is local

Reply:
Ah, okay. So did my change here screw things up?

Reply:
This is the line I edited: `$(MAKE) -C $(BSG_MACHINE_PATH)/.. $(BSG_SIM_EXE)`

Reply:
I think your change is fine.

Reply:
👍